arc-theme-upgrade

```diff
@@ -0,0 +1,20 @@
+#/!bin/bash
+
+# Script to upgrade/install the arc-theme
+
+# Remove previows files
+rm -rf /tmp/arc-theme
+
+git clone https://github.com/horst3180/arc-theme /tmp/arc-theme --depth 1 && cd /tmp/arc-theme
+
+# Remove old versions
+sudo rm -rf /usr/share/themes/{Arc,Arc-Darker,Arc-Dark}
+rm -rf ~/.local/share/themes/{Arc,Arc-Darker,Arc-Dark}
+rm -rf ~/.themes/{Arc,Arc-Darker,Arc-Dark}
+
+#./autogen.sh --prefix=/usr
+./autogen.sh --prefix=/usr --with-gnome=3.16
```

Comment:
`--with-gnome=3.16` will install the wrong theme version on systems with Gnome 3.14 or 3.18, so I would use the commented line above instead.